test/specs/reference/tests/01.reference.js

```diff
@@ -8,20 +8,52 @@ exports.execute = function (options) {
             lowerLimit = 8999,
             upperLimit = 9010;
 
-        var singleEnitityUri = options.url + "/catalog/" + catalog_id + "/entity/"
-            + schemaName + ":" + tableName + "/id=" + entityId;
+        var baseUri = options.url + "/catalog/" + catalog_id + "/entity/"
+            + schemaName + ":" + tableName;
 
-        var multipleEntityUri = options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":"
-            + tableName + "/id::gt::" + lowerLimit + "&id::lt::" + upperLimit;
+        var singleEntityUri = baseUri + "/id=" + entityId;
+
+        var multipleEntityUri = baseUri + "/id::gt::" + lowerLimit + "&id::lt::" + upperLimit;
 
 
         // Test Cases:
+        describe('for a single entity,', function () {
```

Comment:
This implementation is currently tested and working for single entities while I make sure the UI can handle that properly. I can wait to merge this until I clean up `recordedit` and include multiple entities. (It might already work for multiple entities)